Quote links (#100)
oddbird/oddleventy: content/work/adobe.md

---
title: Adobe Fonts

> banner: Adobe Fonts

Reviewer: `banner` is only needed when it's different from the tab `title`

> sub: Adobe Fonts typography superpower game

Reviewer: Why repeat "Adobe Fonts" here? On other projects it looks like our pattern is e.g.

```yaml
title: Workflow Builder
sub: Tools for data analytics
client: Aunalytics
```

So maybe something like…

```yaml
title: Typographic Superpower Game
sub: A meditation on typography
client: Adobe Fonts
```

Or the title could be `Type-Tagging Game`, or something… Just trying to use those three fields more efficiently and consistently.

---
title: Adobe Fonts
banner: Adobe Fonts
sub: Adobe Fonts typography superpower game
logo: adobe
links:
  site: https://game.fonts.adobe.com/
image:
  src: projects/adobegame.jpg
  alt: Illustrated game question about which font is most calligraphic
client: &client Adobe Fonts
date: 2020-10-20
people:
  - &meghan
    name: Meghan Arnold
    title: Sr. Product Marketing Manager
    face: meghan-arnold.jpg
    url: https://twitter.com/MeghanCArnold/status/1318671589618061312
    venue: *client
tasks:
  - User Experience Design
  - Graphic Design
  - Illustration
  - Animation
  - Front-End Development
press:
  - text: |
      This was such a fun project to work on 

>       **with @TypeTasting, @OddBird & my @AdobeFonts** colleagues.

Reviewer: This feels like a strange line to highlight - because it's already clear who we worked with. Why not bold "This was such a fun project to work on"?

    <<: *meghan
    slug: fun
summary: |
  A meditation on typography, 
  Adobe Fonts' Game allows players to choose 
  the fonts that they feel apply best to different scenarios. 
  Players' input helps Adobe Fonts continue 
  to improve on their new browse-by-tags feature. 

>   OddBird worked with Adobe's internal back-end and type teams 
>   to create illustrations for each question. 
>   Using Greensock and CSS, 
>   we animated the illustrations and page transitions. 
>   We integrated numerous fonts and images 
>   while maintaining high performance standards. 
>   The game is responsive 
>   and accessible across screens sizes and browsers.

Reviewer: The wording seems a bit clunky at the end here. Maybe…

```suggestion
  We worked closely with Adobe
  to create animated & interactive illustrations 
  using Greensock, Nuxt, and CSS --
  while maintaining high performance standards,
  and responsive, accessible design.
```
